notsobot/notsobot.py

```diff
@@ -498,6 +502,8 @@ def make_caption_image(b, text, color, font, x, y, is_gif):
                                 new_image.sequence.append(frame)
                             new_image.save(file=final)
                     i.save(file=final)
+                    i.close()
+                    img.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically on the next line.